root/components/home/About.js

```diff
@@ -0,0 +1,93 @@
+/*
+ * @flow strict
+ * Copyright (C) 2022 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+import {FontAwesomeIcon} from '@fortawesome/react-fontawesome';
+// eslint-disable-next-line import/no-unresolved
+import {faCheckCircle} from '@fortawesome/free-solid-svg-icons';
+
+const About = (): React.Element<'section'> => (
+  <section className="about">
+    <div className="container">
+      <div className="row">
+        <div
+          className="col-lg-6"
+        >
+          <h3
+            className="heading"
+          >
+            {l('About')}
+            {' '}
+            <span
+              className="color-purple"
+            >
+              {l('Music')}
+            </span>
+            <span
+              className="color-orange"
+            >
+              {l('Brainz')}
+            </span>
+          </h3>
```

Comment:
I am fine with keeping this as a single section but I do think we can make the Facts section to be dynamic in the coming future. It would be handy to keep it as a separate component in that case.

Reply:
If it's not too big a deal, I think we can stick to the way it is currently to not complicate things.

Reply:
As it is translators will be given the word "About", "Music" and "Brainz" as separate words, so if for example in a language the translation should be in the order "MusicBrainz About", this would break. That's why these things can be complicated :) 

That said, if we want to we can just make the html part of the translatable string, and then let the translators move it around as needed (see the cases where strings already contain `<strong>` or whatever in the index). `<span>` is already in the list of allowed html for this anyway.

Reply:
We can't split the string like this.  I would probably just remove the coloring, to be honest, but if you feel it's important you can do something like this:
```JS
exp.l('About <span class="color-purple">Music</span><span class="color-orange">Brainz</span>')
```
Note that there can be no space between the span tags, so this will likely require an eslint exception for long lines.

Reply:
Done!